### ISSUE TITLE
manage.py harvest doesn't create all tables from database

### ISSUE THREAD

**Issue by nanorepublica** [koterpillar/aloe_django#27]
Hi there,

I am trying to run `manage.py harvest` on our project having created our first feature and the command seems not to be creating all of our tables, specifically the applications/tables managed by South. From a brief follow through of the code it looks like the correct bits of django-nose are being called so I am not entirely sure what the problem is. Currently using 0.0.13 and django 1.6.

Any help would be appreciated

The output is the following:

```
vagrant@precise64:/vagrant$ ./manage.py harvest
nosetests --logging-clear-handlers --detailed-errors --verbosity=1
Creating test database for alias 'default'...
Got an error creating the test database: database "test_vagrant" already exists

Type 'yes' if you would like to try deleting the test database 'test_vagrant', or 'no' to cancel: yes
Destroying old test database 'default'...
/usr/local/lib/python2.7/dist-packages/jsonfield/fields.py:3: DeprecationWarning: django.utils.simplejson is deprecated; use json instead.
  from django.utils import simplejson as json

CommandError: Database test_vagrant couldn't be flushed. Possible reasons:
  * The database isn't running or isn't configured correctly.
  * At least one of the expected database tables doesn't exist.
  * The SQL was invalid.
Hint: Look at the output of 'django-admin.py sqlflush'. That's the SQL this command wasn't able to run.
The full error: relation "robots_url" does not exist
```

**Comment by koterpillar** [comment]
Can you please try the branch from #28 to see if it fixes the issue?

**Comment by nanorepublica** [comment]
Just tried it and that works, thanks for the prompt fix!

**Comment by koterpillar** [comment]
Fix released as 0.0.15.

**Comment by nanorepublica** [comment]
Thanks :+1: :smile:
